docs/conf.py

```diff
@@ -17,60 +17,64 @@
 
 import os
 import sys
-sys.path.insert(0, os.path.abspath('..'))
+
+sys.path.insert(0, os.path.abspath(".."))
```

Comment:
I guess this happened because you ran black on the docs, which is not supposed to happen.
That's fine by me for this *but* this means you did not use the specs for `black` which are in the repository. To run black you need to be inside the repository `nevergrad` but outside the package `nevergrad`.

Please revert all changes that happened to mujoco and photonics, since they are in the exclude list for black (PRs in progress, I would rather not make it more difficult)

Reply:
Done.